tutor_webhook_receiver/plugin.py

```diff
@@ -2,14 +2,44 @@
 import os
 import pkg_resources
 
+from .__about__ import __version__
 
 templates = pkg_resources.resource_filename(
     "tutor_webhook_receiver", "templates"
 )
 
-config = {}
+config = {
+    "defaults": {
+        "VERSION": __version__,
+        "DOCKER_IMAGE": "webhook_receiver/webhook_receiver:latest",
+        "HOST": "webhooks.{{ LMS_HOST }}",
+        "DB_ENGINE": "django.db.backends.mysql",
+        "DB_NAME": "webhook_receiver",
+        "DB_USER": "webhook_receiver01",
+        "DB_PASSWORD": "",
```

Comment:
If I'm not mistaken, Tutor uses hooks to create the MySQL user and database, and it also has a facility to autogenerate passwords. It looks like we still need that implemented here?

Reply:
Yes I'll move them to `config["add"]` and generate them there